### ISSUE TITLE
Not working well with the current version of Haiku

### ISSUE THREAD

**Issue by mounty1** [HaikuArchives/VNCViewer#1]
This doesn't really work with a development pull, at least on the master branch, as at the time of this posting.  First, this trivial patch is required to make it build:

```
diff --git a/source/connectionwindow.rdef b/source/connectionwindow.rdef
index 3728cb5..7ffa0ae 100644
--- a/source/connectionwindow.rdef
+++ b/source/connectionwindow.rdef
@@ -192,7 +192,8 @@ resource(0, "ConnectionWindow") #'ARCV' archive BWindow {
                        "_color" = -656877568,
                        "_color" = -656877568,
                        "_ploc" = point { 160.000, 37.0000 },
-                       #'CSTR' array "_text" =  array {
+#define ceeStr #'CSTR'
+                       ceeStr array "_text" =  array {
                                "Use host:display\ne.g. snoopy:2\n(Display defaults to 0 if not g"
                                "iven)\n"
                        },
```

but even then, (1) the screen isn't fully drawn when a connection is opened and (2) keystrokes appear not to be passed to the server.

**Comment by kenmays** [comment]
Tested patched build on Haiku - built fine on hrev56783 x86 (GCC 11.2.0).

**Comment by diversys** [comment]
Fixed in d53f11b21d08ce99adba612f90c0f1757b39e62b.
